include/posix/pthread_sched.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
```

Comment:
Now that it's 2018, these copyright headers need updating when (significant) changes are made, right?

Reply:
I've personally never received any guidance that this is necessary.